GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -47,14 +47,39 @@ typedef BOOL (*GULRealContinueUserActivityIMP)(
     id, SEL, UIApplication *, NSUserActivity *, void (^)(NSArray *restorableObjects));
 #pragma clang diagnostic pop
 
+typedef void (*GULRealDidRegisterForRemoteNotificationsIMP)(id, SEL, UIApplication *, NSData *);
+
+typedef void (*GULRealDidFailToRegisterForRemoteNotificationsIMP)(id,
+                                                                  SEL,
+                                                                  UIApplication *,
+                                                                  NSError *);
+
+typedef void (*GULRealDidReceiveRemoteNotificationIMP)(id, SEL, UIApplication *, NSDictionary *);
+
+typedef void (*GULRealDidReceiveRemoteNotificationWithCompletionIMP)(
+    id, SEL, UIApplication *, NSDictionary *, void (^)(UIBackgroundFetchResult));
+
 typedef void (^GULAppDelegateInterceptorCallback)(id<UIApplicationDelegate>);
 
 // The strings below are the keys for associated objects.
 static char const *const kGULContinueUserActivityIMPKey = "GUL_continueUserActivityIMP";
 static char const *const kGULHandleBackgroundSessionIMPKey = "GUL_handleBackgroundSessionIMP";
 static char const *const kGULOpenURLOptionsIMPKey = "GUL_openURLOptionsIMP";
+
+#if TARGET_OS_IOS
```

Comment:
Nit: Move this to the end of this list and add a comment mentioning this method is only available on iOS and not on tvOS.